<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyDialog</class>
 <widget class="QDialog" name="ProxyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Only support socks5 ,if use voice proxy must support udp </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>proxy_server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proxy_server"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>proxy_port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proxy_port"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>proxy_username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proxy_username">
       <property name="placeholderText">
        <string>optional</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>proxy_password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proxy_password">
       <property name="placeholderText">
        <string>optional</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>ok</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
